packages/app/src/public-types.ts

```diff
@@ -196,6 +196,12 @@ export interface FirebaseServerAppSettings
    */
   authIdToken?: string;
 
+  /**
+   * An optional App Check token. If provided, the Firebase SDKs that use App Check will utilizze
```

Comment:
Typo "utilize."

Personally I like in lieu, but I'll bet good money our style guide advises something more like "instead of" or "in place of."

Reply:
Fixed. Updated the comments for the new parts again, so it's worth a full review from the begining if you don't mind.